### PR TITLE
chore(root): resolve high vulnerability

### PR DESCRIPTION
**Pull request by bitgoAaron**
- adds resolution for minimist

Ticket: BG-45291

### CODE REVIEW

**Comment by mmcshinsky-bitgo** [comment]
@bitgoAaron  New web-stream-tools types in openpgp@5.2.1 is breaking our implementation of encrypt and the yarn install when the unit-test removes the yarn.lock file. Possibly lock the library from ^5.1.0 to 5.1.0?

...or we could update the functions if we want the newer version.

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "glob": "^7.1.6",
     "husky": "^4.3.0",
     "lerna": "^4.0.0",
+    "minimist": "1.2.6",
```
Comment (tylerlevine) [inline package.json:19]: do we need this in dev deps?

**Review by tylerlevine — CHANGES_REQUESTED** [review]
one small comment

File: package.json (hunk 1 above)
Comment (bitgoAaron) [reply package.json:19]: well it's not used by the app directly, and only exists to satisfy yarn audit...so yes?

**Review by tylerlevine — APPROVED** [review]

**Comment by tylerlevine** [comment]
Ref: https://github.com/substack/minimist/issues/164

**Pull request merged**